polaris.shopify.com/src/components/Markdown/components/Mdx/index.tsx

```diff
@@ -0,0 +1,58 @@
+import {Grid, GridCellProps} from '@shopify/polaris';
+import {SpaceScale} from '@shopify/polaris-tokens';
+
+import {Card} from '../../../Card';
+
+import styles from './Mdx.module.css';
+import {DirectiveCard, DirectiveStatusName} from '../DirectiveCard';
+
+interface MdxCardProps {
+  children: React.ReactNode;
+  gap?: SpaceScale;
+}
+
+export const MdxCard = ({children, gap = '400'}: MdxCardProps) => (
+  <div className={styles.MdxCard}>
+    <Card>
+      <Grid gap={{xs: gap}}>{children}</Grid>
+    </Card>
+  </div>
+);
+
+interface MdxColumnProps {
+  children: React.ReactNode;
+  span?: '1/3' | '2/3' | '1/2' | 'full';
```

Comment:
Appreciate the feedback! I was following the common naming that Tailwind uses for [their width sizing](https://tailwindcss.com/docs/width) and personally like the fraction notation.

Curious what others think and open to changing it to your suggestion if others agree 👍

Reply:
That naming is perfect, that's how we're referencing those in design so it's nice to have a 1:1 with the language we use 😁

Reply:
To be honest, 1/2, 1/3, etc. is more readable for me because that's how I look at design. And I'm pretty sure most designers are more numeric than verbose.

Reply:
With this approach in my head I've been keeping things at max 1 level of nesting, which means that to have a card with a specific grid or column layout we would need each to have a dedicated component.

```
<CardTwoColumns>
    <Do></Do>
    <Dont></Dont>
</CardTwoColumns>
```

This more verbose naming strategy is fine, as long as we keep the card layout selection to the predictable few that were proposed (I think i was told it'd be 4?). I think this is a strong opinion we should establish early: we should have as few layouts as possible, and make sure that our MDX layout components are added intentionally rather than overwhelm the markdown and contributors

Reply:
Created an issue so we don't lose this useful discussion: https://github.com/Shopify/polaris/issues/10980